src/components/info.js

```diff
@@ -4,24 +4,24 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { InfoCircleOutlined } from '@ant-design/icons';
 
-function Info({ children, iconAfterContent, tooltip }) {
+function Info({ children, iconAfterContent, tooltip, onIconClick }) {
+  const icon = (
+    <InfoCircleOutlined
+      className={
+        classNames({
+          'Info-icon': true,
+          'Info-icon--clickable': !!onIconClick,
+          'Info-icon--help': !!tooltip && !onIconClick,
+          'Info-icon--afterContent': !!tooltip && iconAfterContent
+        })
+      }
+      onClick={onIconClick}
+      />
+  );
   return (
     <span className="Info">
       {!!iconAfterContent && children}
-      {!!tooltip && (
-        <Tooltip title={tooltip}>
-          <InfoCircleOutlined
-            className={
-              classNames(
-                'Info-icon',
-                'Info-icon--help',
-                { 'Info-icon--afterContent': iconAfterContent }
-              )
-            }
-            />
-        </Tooltip>
-      )}
-      {!tooltip && <InfoCircleOutlined className="Info-icon" />}
+      {tooltip ? <Tooltip title={tooltip}>{icon}</Tooltip> : icon}
```

Comment:
Honestly I would rather not repurpose this component and use the questionmark icon from editing sections instead (even if it's not as nice of an icon).
I think we should keep constant with patterns patterns we use and so far this info is hover only, while that section icon opens a dialog. And the fact that the helptext has to be "Click here [...]" is an indicator that the user will expect something else.

Reply:
Ok, makes sense. I reverted the `info` component. But I used the AntD question mark ...

![localhost_3000_admin_tab=user-accounts](https://user-images.githubusercontent.com/2676617/231115747-bfe5c2f3-a48a-4fb0-8994-eed1d1302239.png)

..., as the custom one hurts the eye so much:

![localhost_3000_admin_tab=user-accounts (1)](https://user-images.githubusercontent.com/2676617/231115855-9868aa27-9212-48e9-af71-45ef2af4d219.png)

Reply:
Yeah, I know 😓 That one is by far not the best icon we've got, from the custom ones.
The AntD one works well 👍 
As tooltip, maybe we can now simply use the same one as for the document section help, `common.openHelp`, what do you think? Short and to the point, more chances of the text actually being read by the user (and consistent).

Reply:
Definitely :+1: Done!